cdap-api/src/main/java/co/cask/cdap/api/metadata/MetadataEntity.java

```diff
@@ -0,0 +1,212 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package co.cask.cdap.api.metadata;
+
+import co.cask.cdap.api.dataset.lib.KeyValue;
+
+import java.util.Collections;
+import java.util.LinkedList;
+import java.util.List;
+
+/**
+ * Entity representation for Metadata
+ */
+public class MetadataEntity {
+
+  public static final String NAMESPACE = "namespace";
```

Comment:
Why these constants need to be public and in cdap-api? How are they supposed to be used?